deployment/aws-terraform/account-scanner-stack/main.tf

```diff
@@ -0,0 +1,23 @@
+resource "aws_cloudformation_stack" "fss-account-scanner-by-tf" {
+  name = "fss-account-scanner-by-tf"
```

Comment:
For more flexible, you could add a variable such as `CFNStackName` in variables.tf and set a default value `fss-account-scanner-by-tf` for user to to define the CloudFormation stack's name.

Reply:
make sense!